apps/gp2-server/src/app.ts

```diff
@@ -189,7 +215,13 @@ export const appFactory = (libs: Libs = {}): Express => {
   const eventDataProvider =
     libs.eventDataProvider ||
     new EventSquidexDataProvider(eventRestClient, squidexGraphqlClient);
+  const outputDataProvider =
+    libs.outputDataProvider ||
+    new OutputSquidexDataProvider(squidexGraphqlClient, outputRestClient);
 
+  const externalAuthorDataProvider =
+    libs.externalAuthorDataProvider ||
+    new ExternalAuthorSquidexDataProvider(externalAuthorRestClient);
```

Comment:
we don't use the squidexGraphqlClient for this?

Reply:
Because there is only a create method. For creating things we use Rest.

In CRN, External Authors are fetched from Algolia. We need to discuss this, when we need to get them.